taxi_service/settings.py

```diff
@@ -0,0 +1,77 @@
+import os
+from pathlib import Path
+
+
+SECRET_KEY = "django-insecure-012931238jjsoijef91-1-2ead9-0dfus0df9uijeifjsodif9"
+
+DEBUG = True

```

Comment:
The `DEBUG` setting is set to `True`. This is fine for development, but remember to set it to `False` in production to avoid exposing sensitive information.